messaging/src/main/java/org/axonframework/eventhandling/TrackingToken.java

```diff
@@ -56,4 +56,12 @@
      */
     boolean covers(TrackingToken other);
 
+    /**
+     * Return the position this token represents
```

Comment:
I think JavaDoc should clearly state that this is an estimated relative position of the token. For example, GapAwareTrackingToken may have a position, but if there are a lot of gaps, it may need to process some events prior to that position. Two tokens with the same position aren't guaranteed to have the exact same progress.

Reply:
Thanks for the remarks.
I changed it a bit but if you have an exact text you wish to see there I'm willing to copy paste it ;)